declarative-config-bridge/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+  id("otel.nullaway-conventions")
+}
+
+group = "io.opentelemetry.instrumentation"
+
+dependencies {
+  compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
+  implementation("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure-spi")
+  implementation("io.opentelemetry:opentelemetry-api-incubator")
```

Comment:
can you remind me how it depends on opentelemetry-api-incubator?

Reply:
`io.opentelemetry.api.incubator.config.DeclarativeConfigProperties`

Reply:
is it good to merge then @trask ?